concourse/model/traits/update_component_deps.py

```diff
@@ -65,6 +71,15 @@ class UpstreamUpdatePolicy(enum.Enum):
         type=MergePolicy,
         doc='whether or not created PRs should be automatically merged',
     ),
+    AttributeSpec.optional(
+        name='merge_method',
+        default='merge',
```

Comment:
why not use `rebase` as default? 🤔

Reply:
Since we currently use github3.py's default, we use merge. I've opted to keep it that way - can of course be easily changed later.